collect_app/src/main/java/org/odk/collect/android/widgets/TimeWidget.java

```diff
@@ -218,8 +218,7 @@ public void setTitle(CharSequence title) {
          */
         @SuppressWarnings("deprecation")
         private void fixSpinner(Context context, int hourOfDay, int minute, boolean is24HourView) {
-            // android:timePickerMode spinner and clock began in Lollipop
-            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
+            if (Build.VERSION.SDK_INT == Build.VERSION_CODES.N) {
```

Comment:
How did you determine this is now only relevant for N?

I think the block in https://github.com/opendatakit/collect/blob/4eb3b2b1d7be3e62ae249e450c526a0673d5b40b/collect_app/src/main/java/org/odk/collect/android/widgets/TimeWidget.java#L253 should also be updated accordingly since that test doesn't make sense anymore.

Reply:
Yes you are right it's even updated in the gist you have used https://gist.github.com/jeffdgr8/6bc5f990bf0c13a7334ce385d482af9f